Source/WTF/wtf/StdLibExtras.h

```diff
@@ -1091,6 +1091,13 @@ ALWAYS_INLINE constexpr void forEachSetBit(std::span<const WordType> bits, size_
     }
 }
 
+template<typename T, typename U>
+ALWAYS_INLINE void initializeOnce(const std::unique_ptr<T>& ptr, std::unique_ptr<U>&& obj)
+{
+    RELEASE_ASSERT(!ptr);
+    const_cast<std::unique_ptr<T>&>(ptr) = std::move(obj);
```

Comment:
Why aren't we using `WTFMove()` here?

Reply:
Because WTFMove is defined a couple lines beneath in this file.

Reply:
We can always re-order things :)

Reply:
It's not as clean because of the namespace. We use `std::move` elsewhere in this file so it's probably fine. The only difference between `WTFMove` and `std::move` is that WTFMove can catch misuse but here, we know it's not an misuse.